cmd/server/flags.go

```diff
@@ -247,6 +248,12 @@ var flags = append([]cli.Flag{
 		Name:    "config-service-endpoint",
 		Usage:   "url used for calling configuration service endpoint",
 	},
+	&cli.StringFlag{
+		Sources: cli.EnvVars("WOODPECKER_EXTENSIONS_ALLOWED_HOSTS"),
+		Name:    "extensions-allowed-hosts",
+		Usage:   "Hosts that are allowed to be contacted by extensions",
+		Value:   host_matcher.MatchBuiltinExternal,
```

Comment:
as it has security implications e.g. extract secrets ... it should be disabled by default